09-struct.slide

```diff
@@ -2,13 +2,88 @@ Defined types and structs in Go
 22 Feb 2017
 Tags: edmontongo, golang, workshop
 
+
 * Defined types
 
-type keyword. defining a new type
+Types help us organize data.
+
+- give more specific meaning to basic types:
+
+.play -edit 09-struct/conflation/main.go /START BASIC OMIT/,/END BASIC OMIT$/
+
+- create aggregate types:
+
+.play -edit 09-struct/conflation/main.go /START STRUCT OMIT/,/END STRUCT OMIT$/
+
+
+* Defining a new type
+
+A type declaration creates a new named type with the given underlying type. The new type is not an alias of the underlying type.
```

Comment:
This reads like the spec. While that's okay, maybe less formal wording would be easier to follow?

Also, not everything you say needs to be on the slides.